### DISCUSSION TITLE
Expected first argument to collection() to be a CollectionReference, a DocumentReference or FirebaseFirestore

### DISCUSSION THREAD

**Discussion by Tsquared86** [capawesome-team/capacitor-firebase#529] [Q&A]
Why do  I get this error even though my reference is the same format as the online example?

ref"users/5AUE2LQATAeCuJZQ2sKHlv8tvmL2"

   const { snapshot } = await FirebaseFirestore.getDocument({
        reference: ref
      });
      console.log(snapshot)
      return snapshot;

**Comment by robingenz** [comment]
This is not enough information to help you. Please create a bug report and provide a reproduction example.

**Comment by Tsquared86** [comment]
I was importing the new firebase10.7.1 incorrectly some how. After a fresh install and setup everything works as planned. Not an issue with your plugin.
